.github/workflows/docs-pr.yaml

```
@@ -0,0 +1,57 @@
+        uses: actions/checkout@v4.1.7
+        with:
+          # Fetch full history for checking diffs
+          fetch-depth: 0
+
+      - uses: pnpm/action-setup@v4.0.0
+        with:
+          version: 9.14.2
+
+      - name: Get pnpm store directory
+        id: pnpm-cache
+        run: |
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)